src/test/java/teammates/test/cases/webapi/ArchiveCourseActionTest.java

```diff
@@ -28,71 +30,67 @@ protected String getRequestMethod() {
     protected void testExecute() throws Exception {
         InstructorAttributes instructor1OfCourse1 = typicalBundle.instructors.get("instructor1OfCourse1");
         String instructorId = instructor1OfCourse1.googleId;
-        String courseId = instructor1OfCourse1.courseId;
 
         loginAsInstructor(instructorId);
 
         ______TS("Not enough parameters");
         verifyHttpParameterFailure();
-        verifyHttpParameterFailure(Const.ParamsNames.COURSE_ID, courseId);
```

Comment:
Do you have test case somewhere where parameters are there but request body is null?